README.md

```diff
@@ -31,31 +31,31 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. What is the DOM?
-
+DOM, stands for the Document Object Model. This is the thing that gives javascript access to the HTML.
```

Comment:
To expand on this a bit you can add that the DOM is an API through which javascript can be used to manipulate elements on the Window object. The DOM serves as an intermediatry between the window object and the javascript one would like to use to manipulate elements on the window object.